### PR TITLE
[7.0] Update legacy-decorators documentation

### PR DESCRIPTION
**Pull request by xtuc**
| Q                        | A 
| ------------------------ | ---
| Patch: Bug Fix?          | no
| Major: Breaking Change?  | no
| Minor: New Feature?      | no
| Deprecations?            | no
| Spec Compliancy?         | no
| Tests Added/Pass?        | no
| Fixed Tickets            | no
| License                  | MIT
| Doc PR                   | yes
| Dependency Changes       | no

Mentioned in babel/babel#5220.

### CODE REVIEW

File: packages/babel-plugin-transform-decorators/README.md [hunk 1]
```diff
@@ -2,6 +2,8 @@
 
 > Compile class and object decorators to ES5
 
+This plugin is included in `babel-preset-stage-1` (see the [documentation](https://babeljs.io/docs/plugins/preset-stage-1/) for more informations).
```
Comment (existentialism) [inline packages/babel-plugin-transform-decorators/README.md]: Nit: informations -> information

**Comment by hzoo** [comment]
Once we move the new code (legacy decorators code -> this package) in, this is good

File: packages/babel-plugin-transform-decorators/README.md (hunk 1 above)
Comment (xtuc) [inline packages/babel-plugin-transform-decorators/README.md]: Thanks @existentialism

**Comment by hzoo** [comment]
This doesn't seem necessary anymore since we are moving the plugin into the repo now

**Pull request closed without merging**